src/index.js

```diff
@@ -92,6 +92,10 @@ function radredis(schema, transforms, port, host, options){
         .map((objs) => { this.push(objs) })
         .then(() => callback() )
       }))
+    },
+
+    count: () => {
+      return redis.get(`${keyspace}:id`).then(count => parseInt(count, 10))
```

Comment:
We're not deleting at all that I know of, but the capability exists in radredis and it is a hard delete currently